libnymea/interfaces/useraccesscontrol.json

```diff
@@ -0,0 +1,32 @@
+{
+    "extends": "accesscontrol",
+    "states": [
+        {
+            "name": "users",
+            "type": "QStringList"
+        }
+    ],
+    "events": [
+        {
+            "name": "accessGranted",
```

Comment:
Should be named `userAccessGranted`, since we already have this event in `accesscontrol.json`

Reply:
not entirely sure on that... We also do this kind of method overloading in the button interfaces...